File: 백준/Bronze/6825. Body Mass Index/Body Mass Index.ts
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

const weight = parseFloat(input[0]);
const height = parseFloat(input[1]);

const bmi = weight / (height * height);

if (bmi < 18.5) {
    console.log("Underweight");
} else if (bmi <= 25.0) {
    console.log("Normal weight");
} else {
    console.log("Overweight");
}

File: 백준/Bronze/6825. Body Mass Index/Body Mass Index.go
package main

import "fmt"

func main(){
	var weight, height float64
	fmt.Scan(&weight)
	fmt.Scan(&height)

	 bmi := weight / (height * height)

	 if bmi < 18.5 {
		 fmt.Println("Underweight")
	 } else if bmi <= 25.0 {
		 fmt.Println("Normal weight")
	 } else {
		 fmt.Println("Overweight")
	 }
}